### PR TITLE
James develop 22 sep 2020

### PR DESCRIPTION
**Pull request by u2cc**
Add a new script. The intention is both functional and for script sample/knowledge buildup.

### CODE REVIEW

File: MA Crossover and Px Position [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// @version=2
```
Comment (donlingliang) [inline MA Crossover and Px Position:1]: I am fine with `@version2`.
But can we try to stick with `@version 4`? I bet most of the original built-in variables are deprecated in `version 4`.

File: MA Crossover and Px Position [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+// Indicator #1 is an indicator labeling the downward and upward price crossover with fast MA 
+// when the overall MAs are in bullish and bearish pattern respectively
+// Indicator #2 is an indicator for MA crosses. 
+// There is also a background feature to indicate bearish, neutral and bullish period
+
```
Comment (donlingliang) [inline MA Crossover and Px Position:7]: you need to put your name as the author in the script.

**Review by donlingliang — COMMENTED** [review]
It is better to link a video clip URL/documents to this script so that we can give more context or traceback for people, this is something I forgot to do in my pr.

Otherwise LGTM 👍

**Comment by u2cc** [comment]
Like a YouTube URL or text description to this script?

Thx

James

**Comment by donlingliang** [comment]
> Like a YouTube URL or text description to this script? Thx James.

yeah, some references like that, which is good for reviewers. I am thinking if we need a simple pr template as well...

**Pull request closed without merging**